src/OpenColorIO/ops/fixedfunction/FixedFunctionOpCPU.cpp

```diff
@@ -103,6 +103,39 @@ class Renderer_ACES_DarkToDim10_Fwd : public OpCPU
     float m_gamma;
 };
 
+class Renderer_ACES_GamutMap13_Fwd : public OpCPU
+{
+public:
+    Renderer_ACES_GamutMap13_Fwd(ConstFixedFunctionOpDataRcPtr & data);
```

Comment:
I probably c/c from `Renderer_ACES_Glow03_Fwd` so I applied the same recommendation there. Is there a benefit of deleting the default constructor by the way, shouldn't it be already not emitted by the compiler because we define a user constructor ?

Reply:
In that case, you are right the default empty constructor is not generated but the default copy constructor & assignation operator are. So, the code forbids anything not needed by default and that also provides some code consistency i.e. if one of these four is needed then define the four (thanks to C++11 and higher `default` keyword can now be used).  That's an home-made extension of the [rule of three](https://en.cppreference.com/w/cpp/language/rule_of_three). The other side-effect is to disable the move constructor and assignation operator.